packer/templates/vagrant-base-ubuntu-20.04-amd64/template.json

```diff
@@ -74,6 +74,7 @@
       "execute_command": "echo 'vagrant' | {{.Vars}} sudo -S -E sh -eux '{{.Path}}'",
       "expect_disconnect": true,
       "scripts": [
+        "../../scripts/ubuntu/python-is-python3.sh",
```

Comment:
I was thinking on installing `python-is-python3` in those roles where Python is used, as an optional task for Focal. Doing it this way, we'll still need to add it to other playbooks/machines.

Reply:
https://github.com/artefactual-labs/ansible-nginx/pull/13/files#r483948171
https://github.com/artefactual-labs/ansible-atom/pull/55/files#r483948307

Not sure if it may be needed in other places ...

Reply:
I thought perhaps installing that package in the 20.04 VM image would make it available for use in the roles when they are run afterwards?

Reply:
Hi @jraddaoui! - I have removed python-is-python3 from am-packbuild here (see latest commit) and have moved installation of this package to the Ansible roles repos.

Reply:
> I thought perhaps installing that package in the 20.04 VM image would make it available for use in the roles when they are run afterwards?

Makes sense @sbreker, but then it will be needed in the deploy-pub and other playbooks. Doing it in the roles where it's needed should avoid this change in other images/Vagrantfiles/playbooks.